### DISCUSSION TITLE
PinInput does not have a displayable error

### DISCUSSION THREAD

**Discussion by jasonlewicki** [mantinedev/mantine#5267] [Feature requests]
### What package has an issue?

@mantine/form

### Describe the bug

the error prop looks to be unimplemented

### What version of @mantine/* packages do you have in package.json? (Note that all @mantine/* packages must have the same version in order to work correctly)

7.1.5

### Do you know how to fix the issue

No

### Do you want to send a pull request with a fix?

No

**Comment by brumm** [comment]
I noticed this today as well. I expected PinInput to support similar props to other inputs, namely `label`, `description`, `error: ReactNode`
